xdebug.c

```diff
@@ -1191,6 +1203,9 @@ PHP_RINIT_FUNCTION(xdebug)
 	XG(profiler_enabled) = 0;
 	XG(breakpoints_allowed) = 1;
 
+	/* Initialize gc statistics */
+	array_init(&XG(gc_runs));
```

Comment:
`xdebug_llist` actually seems more suited to get this working, I will try again :-)